external-crates/move/crates/move-compiler/tests/move_check/parser/use_inner_scope_invalid.exp

```diff
@@ -7,18 +15,12 @@ error[E01002]: unexpected token
   │         Unexpected 'use'
   │         Expected an expression term
 
-error[E03002]: unbound module
+error[E01002]: unexpected token
   ┌─ tests/move_check/parser/use_inner_scope_invalid.move:6:13
   │
 6 │         use 0x1::M::foo;
-  │             ^^^^^^ Invalid 'use'. Unbound module: '0x1::M'
-
-error[E01002]: unexpected token
-  ┌─ tests/move_check/parser/use_inner_scope_invalid.move:7:9
-  │
-7 │         foo(x)
-  │         ^^^
-  │         │
-  │         Unexpected 'foo'
-  │         Expected a module member: 'spec', 'use', 'friend', 'const', 'fun', or 'struct'
+  │             ^^^
+  │             │
+  │             Unexpected '0x1'
+  │             Expected a module member: 'spec', 'use', 'friend', 'const', 'fun', or 'struct'
```

Comment:
Another situation where we should consider doing something to move past the misplaced `use` (like `friend` above).